### ISSUE TITLE
Minor java doc error

### ISSUE THREAD

**Issue by joa23** [OpenHFT/Chronicle-Bytes#82]
```
  /**
     * Allocate an elastic buffer with initially no size.
     *
     * @return Bytes for writing.
     */
    static NativeBytes<Void> allocateElasticDirect(long initialCapacity) throws IllegalArgumentException {
        return NativeBytes.nativeBytes(initialCapacity);
    }
```

I assume this method actually allocates a buffer with initialCapacity..

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.5](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.5), [BOM-2.17.72](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.72)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.5](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.5), [BOM-2.17.74](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.74)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
